qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -88,7 +88,12 @@ def __init__(
 
         # Set configurable options
         self.set_experiment_options(lengths=list(lengths), num_samples=num_samples)
-        self.set_analysis_options(data_processor=probability(outcome="0" * self.num_qubits))
+        self.set_analysis_options(
```

Comment:
Should we use the get function of the data processor library?

Reply:
I checked but we cannot use the getter. This is a custom processor for RB, because outcome depends on number of qubits, and it counts for label "0". Of course we can modify the getter, but this should be done in the separate PR.